qa/compare_snapshots.py

```diff
@@ -0,0 +1,38 @@
+import click
+import requests
+
+from qa.comparitor import Comparitor
+
+@click.command("compare-snapshots")
+@click.option(
+    "--snapshot1",
+    '-s1',
+    type=int,
+    help="The number of the snapshot to compare",
+    required=True,
+)
+@click.option(
+    "--snapshot2",
+    '-s2',
+    type=int,
+    help="The number of the snapshot to compare with",
+    required=True,
+)
+@click.option(
+    "--intervention",
+    type=str,
```

Comment:
I think you can make this `type=Intervention` (though it might be some variation on it), and you can add a default:
`default=Intervention.SELECTED_INTERVENTION`